e2e/test/iothub/messaging/MessageReceiveFaultInjectionTests.cs

```diff
@@ -397,8 +397,6 @@ async Task InitOperationAsync(IotHubDeviceClient deviceClient, TestDevice testDe
                         // ignore and proceed.
                     }
                 }
-
-                await serviceClient.Messaging.CloseAsync().ConfigureAwait(false);
```

Comment:
@timtay-microsoft I see that this was added in #2614 . The service client instance in this test is used throughout the lifetime of the test, so I've removed this "close" call from the "InitOperationAsync" task and left it only in the "CleanupOperationAsync" task. Let me know if there is some other reason why this is required in the init operation block.